TradeItIosTicketSDK2/TradeItTransactionDetailsViewController.swift

```diff
@@ -0,0 +1,70 @@
+import UIKit
+
+class TradeItTransactionDetailsViewController: TradeItViewController, UITableViewDelegate, UITableViewDataSource {
+
+    var transaction: TradeItTransaction?
+    var accountBaseCurrency: String?
+
+    @IBOutlet weak var transactionDetailTableView: UITableView!
+
+    private var transactionDetailsRows = [TransactionDetailsRow]()
+    private var presenter: TradeItTransactionPresenter?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let transaction = self.transaction else {
+            preconditionFailure("TradeItIosTicketSDK ERROR: TradeItTransactionDetailsViewController loaded without setting transaction.")
+        }
+        guard let accountBaseCurrency = self.accountBaseCurrency else {
+            preconditionFailure("TradeItIosTicketSDK ERROR: TradeItTransactionDetailsViewController loaded without setting accountBaseCurrency.")
+        }
+        self.transactionDetailTableView.delegate = self
+        self.transactionDetailTableView.dataSource = self
+        self.presenter = TradeItTransactionPresenter(transaction, currencyCode: accountBaseCurrency)
+        self.title = self.presenter?.getTransactionTypeLabel()
+        if self.presenter?.getTransactionDescriptionLabel() != "" {
```

Comment:
@guillaumedebavelaere what if this returns null? It will still add the row. Is that intended?